CHANGELOG.md

```diff
@@ -4,6 +4,36 @@ All notable changes to Merlin will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.4]
+
+### Added
+- cli test flag `--local`, which can be used in place of listing out the id of each
```

Comment:
since these changes add features to the API we should bump to 1.3.0

Reply:
Bumped to 1.3.0